concepts/observability-strategy/strategy.md

```diff
@@ -1,54 +1,60 @@
+2. Collection and preprocessing of the signals from the user application and the surrounding infrastructure, including metadata enrichment
+3. Delivery of the signal to an backend for storage
+4. Optionally, aggregation of the signals
+5. Storage of the signals
+6. Analysis, querying, and dashboarding of the signals
 
-Providing ready-to-use in-cluster backends is necessarily opinionated, does not cover all usage scenarios, and does not fit into Kyma's goal of providing the Kubernetes building blocks to integrate into the the SAP ecosystem. Also, the licensing issues (particularly with Grafana and Loki, or ElasticSearch as an alternative backend technology) show that an opinionated stack is problematic. It's better to handle opinionation by integrating with actual managed services. 
+![a](./assets/strategy-stages.drawio.svg)
 
-The following strategy proposes a new approach for the Kyma observability stack by opening up to new scenarios by supporting convenient integration at runtime. At the same time, it reduces the focus on ready-to-use solutions running inside the cluster.
+The current Kyma observability stack is spreading over all stages and is providing a lightweight end-to-end setup for basic needs. Hereby, configuration possibilities are available at installation time mainly.
 
-## Goals
+![Current observability stack](./assets/strategy-current.drawio.svg "Current observability stack")
 
-### Stages of observability
+- Application Logs
+  - Instrumentation is done by printing logs to stdout/stderr as recommended by Kubernetes best practices, system components are following that guide and logs are available already; no other way of getting logs collected
+  - FluentBit as log collector defines a hardcoded pipeline for collecting, enriching and pushing the logs to a backend; no further configuration of pipelines at runtime, especially additional outputs for external systems cannot be configured
+  - Storage is realized by a lightweight (non-scalable) Loki installation, no configuration at runtime
+  - Reporting is implemented by the log explorer of Grafana
+- Metrics
+  - For instrumentation workload needs to expose metrics in the prometheus-compatible format, system components are already doing that; no other ways of exporting metrics is supported like using the OTLP push based protocol
+  - Collection of metrics from system and custom workloads is done by a lightweight prometheus installation; configuration of collection can be defined at runtime but the storage might not scale in accordingly requiring adjustments at deploy time; configuration of outputs is not possible at runtime and only prometheus-specific protocols are supported (forward or federation)
+  - Prometheus is collecting and storing the metrics; setup is non-scalable and resource settings cannot be configured at runtime
+  - Reporting is done via a Grafana installation which loads pre-bundled dashboards; dashboards can be added at runtime
+- Traces
```

Comment:
Consider formatting this as h3 and reducing the indentation for the sub-bullets.


```suggestion
### Traces
```